### PR TITLE
Pull Request for Issue 2456: Change region start and end labels.

### PR DESCRIPTION
**Pull request by mButlerDM**
Modified AMStepScanAxisView to allow for setting region labels at initialization. 

Used new functionality to change labels in PGM to Min and Max, instead of Start and End, respectively.

### CODE REVIEW

**Comment by mButlerDM** [comment]
@XuetaoLiu-CLS Could you review when you have a moment? Thanks!

File: source/ui/dataman/AMStepScanAxisView.h [hunk 1]
```diff
@@ -39,7 +39,7 @@ class AMStepScanAxisElementView : public QWidget
 
 public:
 	/// Constructor.  Builds a view for a single region.
-	explicit AMStepScanAxisElementView(AMScanAxisRegion *region, QWidget *parent = 0);
+	explicit AMStepScanAxisElementView(AMScanAxisRegion *region, QWidget *parent = 0, QString startLabel = "", QString endLabel = "");
```
Comment (XuetaoLiu-CLS) [inline source/ui/dataman/AMStepScanAxisView.h]: should we default this to `Start` and `End` as well?

**Comment by XuetaoLiu-CLS** [comment]
Look good other than a question on default label value

Time Estimate: 1 minute

File: source/ui/dataman/AMStepScanAxisView.h (hunk 1 above)
Comment (dretrex) [inline source/ui/dataman/AMStepScanAxisView.h]: Yes.  Also the arguments should be `const QString &`.

**Comment by mButlerDM** [comment]
@dretrex Made those suggested changes.

**Comment by dretrex** [comment]
Looks good.
Time Estimate: 1 minute

**Pull request merged**